src/audio/volume/volume.c

```diff
@@ -1213,6 +1211,51 @@ static vol_zc_func vol_get_zc_function(struct comp_dev *dev)
 	return NULL;
 }
 
+/**
+ * \brief Set volume frames alignment limit.
+ * \param[in,out] source Source structure.
+ * \param[in,out] sink Sink structure.
+ */
+#if XCHAL_HAVE_HIFI3 || XCHAL_HAVE_HIFI4
```

Comment:
is there any harm in just always forcing alignment to optimize code? Or is this part of a phased transition?